src/odemis/gui/cont/acquisition/fastem_acq.py

```
@@ -819,13 +887,14 @@ def set_next_toa_settings(self, _, next_toa_data):
             logging.exception("Failed to set next TOA settings")
 
     @call_in_wx_main
-    def on_toa_acquisition_done(self, future, toa, window):
+    def on_toa_acquisition_done(self, future, toa, window, save_dir):
         """
         Callback called when a TOA acquisition is finished (either successfully,
         cancelled or failed)
         :future: (ProgressiveFuture) the future of the acquisition.
         :param toa: (FastEMTOA) the TOA object.
         :param window: (NodeWindow) the window of the TOA.
+        :param save_dir: (str) directory where this TOA’s images are stored.
```

Ruff (0.14.11)

Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)